src/components/Button/Button.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+import styles from './Button.module.css';
+
+const isOperator = val => {
+    return !isNaN(val) || val === "." || val === "=";
+}
+
+const Button = ({ content, handleClick }) => {
+    const isoperator = isOperator(content) ? false : true;
```

Comment:
I think it should be isNotOperator, because you invert logic here.
Moreover it should be done in function itself, then you can use it in `[styles.operator]: isOperator(content)`